### PR TITLE
refactor(components): [popper] use JSX in Unit test

### PR DESCRIPTION
**Pull request by holazz**
https://github.com/element-plus/element-plus/issues/6114

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by holazz** [comment]
cc @sxzz

File: packages/components/popper/src/content.ts [hunk 1]
```diff
@@ -103,3 +104,5 @@ export type UsePopperContentProps = ExtractPropTypes<
 export type UsePopperCoreConfigProps = ExtractPropTypes<
   typeof usePopperCoreConfigProps
 >
+
+export type PopperContentInstance = InstanceType<typeof Content>
```
Comment (ryuhangyeong) [inline packages/components/popper/src/content.ts:108]: Hello! if you look at the component/popper/index.ts file, there is a source code that performs the same role as this syntax.

looks like one of the two needs to be fixed.


```ts
import { withInstall } from '@element-plus/utils'
import Popper from './src/popper.vue'

import ElPopperArrow from './src/arrow.vue'
import ElPopperTrigger from './src/trigger.vue'
import ElPopperContent from './src/content.vue'

export { ElPopperArrow, ElPopperTrigger, ElPopperContent }

export const ElPopper = withInstall(Popper)
export default ElPopper

export * from './src/popper'
export * from './src/trigger'
export * from './src/content'
export * from './src/arrow'

export type { Placement, Options } from '@popperjs/core'
export type ElPopperArrowInstance = InstanceType<typeof ElPopperArrow> // here
export type ElPopperArrowTrigger = InstanceType<typeof ElPopperTrigger> // here
export type ElPopperArrowContent = InstanceType<typeof ElPopperContent> // here
```

File: packages/components/popper/src/trigger.ts [hunk 2]
```diff
@@ -18,3 +20,5 @@ export const usePopperTriggerProps = buildProps({
 } as const)
 
 export type PopperTriggerProps = typeof usePopperTriggerProps
+
+export type PopperTriggerInstance = InstanceType<typeof Trigger>
```
Comment (ryuhangyeong) [inline packages/components/popper/src/trigger.ts:24]: same here

File: packages/components/popper/src/arrow.ts [hunk 3]
```diff
@@ -10,3 +11,5 @@ export const usePopperArrowProps = buildProps({
 } as const)
 
 export type UsePopperArrowProps = ExtractPropTypes<typeof usePopperArrowProps>
+
+export type PopperArrowInstance = InstanceType<typeof Arrow>
```
Comment (ryuhangyeong) [inline packages/components/popper/src/arrow.ts:15]: same here

File: packages/components/popper/src/content.ts (hunk 1 above)
Comment (holazz) [reply packages/components/popper/src/content.ts:108]: Thanks for comments, I have refactor it in another [PR](https://github.com/element-plus/element-plus/pull/8529)

**Review by sxzz — APPROVED** [review]

**Pull request merged**